src/System.Collections.Concurrent/src/System/Collections/Concurrent/PartitionerStatic.cs

```diff
@@ -899,14 +898,12 @@ private class InternalPartitionEnumerator : DynamicPartitionEnumerator_Abstract<
                     IEnumerator<TSource> sharedReader,
                     SharedLong sharedIndex,
                     SharedBool hasNoElementsLeft,
-                    object sharedLock,
```

Comment:
I assume this sharedLock was just dead code but have we verified that it is intentionally dead and not just a bad port?